styleguide/source/_patterns/02-molecules/tools-list.twig

```diff
@@ -0,0 +1,7 @@
+<ul class="ama__tools--list">
+  {% for tool in tools %}
+    <li class="ama__tools--list__tools">
```

Comment:
Since tools-list is it's own pattern it's probably safe to consider it the block level element in these cases in which I'd just use a single hyphen between tools and list (e.g. `ama__tools-list`)